src/keys.rs

```diff
@@ -115,6 +115,7 @@ pub async fn handle_key_bindings(
 
         (MenuItem::Standings, Char('j') | KeyCode::Down, _) => guard.state.standings.next(),
         (MenuItem::Standings, Char('k') | KeyCode::Up, _) => guard.state.standings.previous(),
+        (MenuItem::Standings, KeyCode::Tab, _) => guard.state.standings.toggle_view_mode(),
```

Comment:
I would prefer to not use Tab for this because I think that's a key that could be used in the future for TUI wide control (e.g. switching the tabs). Maybe `l` for "league"? Open to other ideas too

Reply:
Would you want 'l' to toggle back and forth, or 'l' for League and 'd' for Division?

I don't mind either.

Reply:
I think just toggle is fine. 'd' is used for "Debug" already anyway